README.md

```diff
@@ -41,11 +42,11 @@ To create a migration, execute `db-migrate create` with a title. `node-db-migrat
 
 ```javascript
 exports.up = function (db, callback) {
-  callback();
```

Comment:
Right but when you generate the migrations with `db-migrate create add-pets`, it doesn't contain the callback so i thought the docs should reflect what is actually generated. Maybe instead of updating the docs the generated code should be updated to contain `callback();`?